### PR TITLE
Add myself to admins

### PR DESCRIPTION
**Pull request by jimwins**

### CODE REVIEW

**Review by cmb69 — APPROVED** [review]
Makes sense to me.

**Comment by sy-records** [comment]
Hi, @cmb69 I have a question, what are the permissions of the admins here?

**Comment by saundefined** [comment]
@sy-records if I remember correctly, admins grant access to users to create RFCs.

File: dokuwiki/conf/local.protected.php [hunk 1]
```diff
@@ -6,6 +6,7 @@
     'derick',
     'girgias',
     'ilutov',
+    'jimw',
     'kelunik',
     'krakjoe',
     'levim',
```
Comment (sy-records) [inline dokuwiki/conf/local.protected.php:12]:
```suggestion
    'levim',
    'lufei',
```

Add one for me 😃

Comment (cmb69) [reply dokuwiki/conf/local.protected.php:12]: Makes sense to me.

**Review by sy-records — APPROVED** [review]

**Comment by kelunik** [comment]
@cmb69 Why can't they grant karma in practice?

**Comment by cmb69** [comment]
@kelunik, oops! That was a typo; I've fixed it.

**Comment by jimwins** [comment]
Could someone go ahead and merge this? (I don't have commit access to any of the web repos, so I can't.)

**Comment by saundefined** [comment]
Done, but as far as I remember, there's no auto-deploy here

**Comment by php-pulls** [comment]
I'll have a look tomorrow

**Comment by derickr** [comment]
This is now deployed (and the wiki software upgraded), but it would be better if the commit messages included the names of the people who were added, like "Added jimw and lufei" as admins.

**Pull request merged**